### PR TITLE
Fixes tests from #640.

### PR DESCRIPTION
**Pull request by ashfurrow**
Mostly straightforward, except Stripe's SDK folks decided to get fancy with `__attribute__` so I had to get fancy with protocols ¯\\_(ツ)_/¯

Updated screenshots are caused by a fixed bug in the Action library and are expected (notice the corrected button enabled states).

### CODE REVIEW

**Comment by ashfurrow** [comment]
Oops, thought this was #643. Feel free to follow-up with comments @orta.

File: Kiosk/Bid Fulfillment/StripeManager.swift [hunk 1]
```diff
@@ -2,10 +2,28 @@ import Foundation
...
+protocol Clientable {
+    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?)
+}
+extension STPAPIClient: Clientable {
+    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?) {
+        self.createToken(withCard: card) { (token, error) in
+            completion?(token, error)
+        }
+    }
+}
```
Comment (orta) [inline Kiosk/Bid Fulfillment/StripeManager.swift]: hah, yep that is a bit funky

File: KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift [hunk 2]
```diff
@@ -5,19 +5,19 @@ import Keys
 import Kiosk
 
 class StubKeys: EidolonKeys {
-    override func cardflightProductionAPIClientKey() -> String! {
+    override var cardflightProductionAPIClientKey: String {
```
Comment (orta) [inline KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift]: this is cocoapods keys 2.0 but I don;t see a gemfile update?

Comment (ashfurrow) [inline KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift]: Yup, the update was in #640: https://github.com/artsy/eidolon/pull/640/files#diff-e79a60dc6b85309ae70a6ea8261eaf95L48 That PR missed the unit testing target, which this PR addresses.

Comment (orta) [inline KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift]: cool cool

**Pull request merged**